rdata/tests/test_rdata.py

```diff
@@ -1,5 +1,6 @@
 """Tests of parsing and conversion."""
```

Comment:
I agree that would be useful. Thank you for taking care of that! I think it could also be useful to include in the repo an R script that generates all those test data files.

What I was aiming for with this test was to make it more unit-test-like, covering only the functionality provided in `_ascii.py` as most of the parsing code (and all conversion code) is shared between ASCII and XDR files (and covered by previous tests already). But it might well be that there is still some special cases that the current implementation doesn't capture correctly (similar to the oddity with NAs for example).

Reply:
The R script sounds useful, indeed. Lets make an issue out of this.